src/Calendar.js

```diff
@@ -693,10 +699,16 @@ class Calendar extends React.Component {
   };
 
   handleDrillDown = (date, view) => {
-    if (view)
-      this.handleViewChange(view)
+    const { onDrillDown } = this.props;
+    if (onDrillDown) {
+      onDrillDown(date, view, this.drilldownView)
```

Comment:
`return` early here then you can avoid the `else` block around the rest of the function

Reply:
@jquense thanks for a quick response and review =) Done